I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 34, in script
    pause
  File "renpy/common/000statements.rpy", line 416, in execute_pause
    renpy.pause()
  File "game/myScreens.rpy", line 5, in execute
    screen inventory_screen:
  File "game/myScreens.rpy", line 5, in execute
    screen inventory_screen:
  File "game/myScreens.rpy", line 13, in execute
    add inventory.itemNumber
Exception: Not a displayable: 3

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 34, in script
    pause
  File "/Applications/renpy-7.3.5-sdk/renpy/ast.py", line 1949, in execute
    self.call("execute")
  File "/Applications/renpy-7.3.5-sdk/renpy/ast.py", line 1937, in call
    return renpy.statements.call(method, parsed, *args, **kwargs)
  File "/Applications/renpy-7.3.5-sdk/renpy/statements.py", line 277, in call
    return method(parsed, *args, **kwargs)
  File "renpy/common/000statements.rpy", line 416, in execute_pause
    renpy.pause()
  File "/Applications/renpy-7.3.5-sdk/renpy/exports.py", line 1480, in pause
    rv = renpy.ui.interact(mouse='pause', type='pause', roll_forward=roll_forward)
  File "/Applications/renpy-7.3.5-sdk/renpy/ui.py", line 297, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "/Applications/renpy-7.3.5-sdk/renpy/display/core.py", line 2702, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, **kwargs)
  File "/Applications/renpy-7.3.5-sdk/renpy/display/core.py", line 3094, in interact_core
    root_widget.visit_all(lambda i : i.per_interact())
  File "/Applications/renpy-7.3.5-sdk/renpy/display/core.py", line 541, in visit_all
    d.visit_all(callback, seen)
  File "/Applications/renpy-7.3.5-sdk/renpy/display/core.py", line 541, in visit_all
    d.visit_all(callback, seen)
  File "/Applications/renpy-7.3.5-sdk/renpy/display/core.py", line 541, in visit_all
    d.visit_all(callback, seen)
  File "/Applications/renpy-7.3.5-sdk/renpy/display/screen.py", line 430, in visit_all
    callback(self)
  File "/Applications/renpy-7.3.5-sdk/renpy/display/core.py", line 3094, in <lambda>
    root_widget.visit_all(lambda i : i.per_interact())
  File "/Applications/renpy-7.3.5-sdk/renpy/display/screen.py", line 440, in per_interact
    self.update()
  File "/Applications/renpy-7.3.5-sdk/renpy/display/screen.py", line 625, in update
    self.screen.function(**self.scope)
  File "game/myScreens.rpy", line 5, in execute
    screen inventory_screen:
  File "game/myScreens.rpy", line 5, in execute
    screen inventory_screen:
  File "game/myScreens.rpy", line 13, in execute
    add inventory.itemNumber
  File "/Applications/renpy-7.3.5-sdk/renpy/sl2/sldisplayables.py", line 430, in sl2add
    d = renpy.easy.displayable(d, scope=scope)
  File "/Applications/renpy-7.3.5-sdk/renpy/easy.py", line 136, in displayable
    raise Exception("Not a displayable: %r" % (d,))
Exception: Not a displayable: 3

Darwin-19.4.0-x86_64-i386-64bit
Ren'Py 7.3.5.606
Demo 1.0
Tue May 19 23:42:10 2020
